### PR TITLE
Fix audio cls error

### PR DESCRIPTION
**Pull request by HDThang**
# What does this PR do?
-Fix audio classification runtime error require padding=True
-fix mlflow run error for speech-pretraining

### CODE REVIEW

File: examples/pytorch/speech-pretraining/run_wav2vec2_pretraining_no_trainer.py
Comment (yeomni) [inline examples/pytorch/speech-pretraining/run_wav2vec2_pretraining_no_trainer.py:1]: Do we have to run this without Trainer?

File: examples/pytorch/audio-classification/run_audio_classification.py [hunk 1]
```diff
@@ -332,7 +332,7 @@ def train_transforms(batch):
                 audio["array"], max_length=data_args.max_length_seconds, sample_rate=feature_extractor.sampling_rate
             )
             subsampled_wavs.append(wav)
-        inputs = feature_extractor(subsampled_wavs, sampling_rate=feature_extractor.sampling_rate)
+        inputs = feature_extractor(subsampled_wavs, sampling_rate=feature_extractor.sampling_rate, max_length=16000, truncation=True, padding=True)
```
Comment (yeomni) [inline examples/pytorch/audio-classification/run_audio_classification.py:335]: Does running on A100 require the same change? If not, this issue should also be reported.

Comment (HDThang) [reply examples/pytorch/audio-classification/run_audio_classification.py:335]: A100 requires the same change to run normally so I think this is not the maf issues.

**Review by thuc-moreh — APPROVED** [review]
LGTM
